apache/netbeans
# [NETBEANS-5111] - Update JAXB from 2.2 to 2.3

Notes:
- Utilize `jakarta.activation` instead of `javax.activation`
- Utilize `jakarta.xml.bind-api` instead of `jaxb-api`
- Second release under the Eclipse Foundation
- Update license type to EDL-1.0
- Several bug fixes and improvements
- Many Security fixes
- JAXB api can now be loaded as a module
- [Removed](https://javaee.github.io/jaxb-v2/doc/user-guide/ch02.html#a-2-3-0-1) legacy technology dependency `jaxb1-impl`

Testing:
- Full build done
- Verify successfull execution of unit tests for modules `/ide/libs.jaxb` & `/ide/xml.jaxb.api`
- Verify successfull execution of `ant verify-libs-and-licenses`
- Successfully create project that bind WSDL to Java with JAXB as per [NetBeans Documentation](https://netbeans.org/kb/74/websvc/jaxb.html)
- Verify that the files generated used the Eclipse implementation of JAXB

![NETBEANS-5111-1](https://user-images.githubusercontent.com/9832133/101712093-b74c5e80-3a8c-11eb-8b3a-558f1062a031.png)

[JAXB Web Page](https://projects.eclipse.org/projects/ee4j.jaxb-impl)
[Release Notes](https://javaee.github.io/jaxb-v2/doc/user-guide/ch02.html)
